src/impl/dtlssrtptransport.cpp

```diff
@@ -218,33 +218,51 @@ bool DtlsSrtpTransport::demuxMessage(message_ptr message) {
 	}
 }
 
+EncryptionParams DtlsSrtpTransport::getEnctyptionParams(const char *suite) {
+	if (strcmp(suite, "SRTP_AES128_CM_SHA1_80")==0)
+		return {SRTP_AES_128_KEY_LEN, SRTP_SALT_LEN, srtp_profile_aes128_cm_sha1_80};
+	if (strcmp(suite,"SRTP_AES128_CM_SHA1_32")==0)
+		return {SRTP_AES_128_KEY_LEN, SRTP_SALT_LEN, srtp_profile_aes128_cm_sha1_32};
+	if (strcmp(suite,"SRTP_AEAD_AES_128_GCM")==0)
+		return {SRTP_AES_128_KEY_LEN, SRTP_AEAD_SALT_LEN, srtp_profile_aead_aes_128_gcm};
+	if (strcmp(suite,"SRTP_AEAD_AES_256_GCM")==0)
+		return {SRTP_AES_256_KEY_LEN, SRTP_AEAD_SALT_LEN, srtp_profile_aead_aes_256_gcm};
+	throw std::logic_error("Unexpected SRTP suite name: " + std::string(suite));
+}
```

Comment:
You should pass a `string` and compare it simply with the `==` operator, IMO `strcmp` should be reserved to C code.
```suggestion
EncryptionParams DtlsSrtpTransport::getEncryptionParams(string suite) {
	if (suite == "SRTP_AES128_CM_SHA1_80")
		return {SRTP_AES_128_KEY_LEN, SRTP_SALT_LEN, srtp_profile_aes128_cm_sha1_80};
	if (suite == "SRTP_AES128_CM_SHA1_32")
		return {SRTP_AES_128_KEY_LEN, SRTP_SALT_LEN, srtp_profile_aes128_cm_sha1_32};
	if (suite == "SRTP_AEAD_AES_128_GCM")
		return {SRTP_AES_128_KEY_LEN, SRTP_AEAD_SALT_LEN, srtp_profile_aead_aes_128_gcm};
	if (suite == "SRTP_AEAD_AES_256_GCM")
		return {SRTP_AES_256_KEY_LEN, SRTP_AEAD_SALT_LEN, srtp_profile_aead_aes_256_gcm};
	throw std::logic_error("Unexpected SRTP suite name: " + suite);
}
```